packages/salesforcedx-vscode-soql/package.json

```diff
@@ -52,6 +52,7 @@
     "eslint-plugin-jsdoc": "^30.0.3",
     "eslint-plugin-prettier": "^3.1.4",
     "mocha": "^5",
+    "node-prune": "^1.0.2",
```

Comment:
If you are only using this for the `prevscode:package` script I think you can remove it since you are running it using `npx`

Reply:
Yes, thanks.

Reply:
I wasn't sure the protocol here.  If i use it in a script, it's kinda sneaking in dependencies.